app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,46 @@
+ <section class="spotlight">
+    <% top_media = Work.top_media %>
```

Comment:
Hm, instead of setting this local variable here, consider setting it in the `HomepagesController#index` action as an instance variable